chrome/skin/default/zotero/zotero.css

```diff
@@ -430,6 +430,14 @@ label.zotero-text-link {
 	max-width: 29.5em;
 }
 
+.zotero-advanced-options>.zotero-advanced-options-label>dropmarker {
+	list-style-image: url("chrome://browser/skin/toolbarbutton-dropdown-arrow.png");
```

Comment:
Oh, part of the issue with the blurriness may be that this — you generally can't actually rely on browser images, because they're not the same cross-platform. And the dropmarker seems to have a default image anyway. If I disable this rule, the same icon remains but the blurriness goes away. (It still shifts on rotate, though.)